src/app/components/core/hero/hero.component.html

```diff
@@ -1,4 +1,6 @@
 <section class="hero">
   <h1>Административен панел</h1>
-  <img src="../../../assets/icons/hero-wheels.png" alt="asdasd" />
+  <div class="image-container">
+    <img src="../../../assets/icons/hero-wheels.png" alt="asdasd" />
```

Comment:
alt="asdasd" ? And why there is an image in "icons" folder?

Reply:
fixed